docs/Addons.md

```diff
@@ -18,86 +18,33 @@ The [Prow Operator Test] is a good example of a [Basic operator test]. It verifi
 
-[Here](https://gitlab.cee.redhat.com/service/ocm-resources/-/blob/master/data/uhc-production/orgs/13215750.yaml#L13) is an example of SD-CICD's quota for production. The `rh_org_id` (which is also the file name) can be determined by logging into OCM via the `ocm` CLI with your OCM token and running `ocm whoami`.
-
-An example production quota for an addon would look like:
-```
----
-$schema: /org-1.yaml
-
-name: "Your Org Name"
+[Here](https://gitlab.cee.redhat.com/service/ocm-resources/-/blob/master/data/uhc-production/orgs/13215750.yaml#L13) is an example of SD-CICD's quota for production.
 
-rh_org_id: <your-org-id>
+You need to open an MR to update the `SDCICD` org's quota so that it can provision your addon (as well as bumping the number of CCS clusters by 2 or so). You'll need to modify the following three files:
 
-SKUs:
-  # e2e testing for OSD (non CCS)
-  # m5.xlarge singleAZ 4 compute
-  MCT3326: 2
+- [Our production quota](https://gitlab.cee.redhat.com/service/ocm-resources/-/blob/master/data/uhc-production/orgs/13215750.yaml)
+- [Our stage quota](https://gitlab.cee.redhat.com/service/ocm-resources/-/blob/master/data/uhc-stage/orgs/13215750.yaml)
+- [Our integration quota](https://gitlab.cee.redhat.com/service/ocm-resources/-/blob/master/data/uhc-integration/orgs/13215750.yaml)
 
-  <your-addon-sku>: 2
+Please bump the quota for SKU `MW00530` by 2 so that we can provision additional CCS clusters for you!
 
-environment: "uhc-production"
-```
-
-For `int` and `stage`, the file will need to go into a different folder and use a different value for the `environment` key.
-
-> *NOTE*: The SKU `MCT3326` is the SKU that `osde2e` uses to provision OpenShift clusters by default. If you do not request quota of this SKU, your jobs will all fail to provision clusters.
-
-If you want to test on CCS clusters instead, you need to use the SKU `MW00530` and follow the instructions in [CCS Cluster Testing](ccs-cluster-testing).
-
-## OCM Permissions
-
-The OCM user/organization that runs your tests will need some specific permissions within OCM in order to launch your test clusters.
-
-These permissions are configured via [`ocm-resources`](https://gitlab.cee.redhat.com/service/ocm-resources), and will look like this:
-
-```yaml
----
-$schema: /user-1.yaml
-
-user_id: "your-ocm-user"
+### Providing Secrets to Your Build
 
-kerberos_id: "N/A"
+Follow the documentation [here](https://docs.ci.openshift.org/docs/how-tos/adding-a-new-secret-to-ci/) to create secrets and configure them to be mirrored into the `ci` namespace [like ours](https://github.com/openshift/release/blob/master/core-services/secret-mirroring/_mapping.yaml#L62).
```

Comment:
Is there a naming convention for this secret?

Reply:
Ehh, not really. Most people use `<addon-name>-credentials`